### ISSUE TITLE
Visual Bell

### ISSUE THREAD

**Issue by pelillian** [zeit/hyper#3427]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: MacOS 10.14.2
- **Hyper.app version**: 2.1.1
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: 

## Issue

Feature Request:
The regular terminal allows a visual bell instead of an audible one. This is a basic feature that hyper lacks.

**Comment by federicocandiago** [comment]
I agree with OP. It would be absolutely necessary for some of my use cases.
